scheduler/server/load_based_sched_alg.go

```diff
@@ -0,0 +1,781 @@
+package server
+
+import (
+	"fmt"
+	"math"
+	"regexp"
+	"sort"
+	"sync"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/twitter/scoot/common/stats"
+)
+
+const (
+	under int = iota
+	over
+)
+
+// defaults for the LoadBasedScheduler algorithm: only one class and all jobs map to that class
+var (
+	DefaultLoadBasedSchedulerClassPercents = map[string]int32{"c0": 100}
```

Comment:
Realizing now, we are making the scheduler more stateful. When schedulers restart they will lose any configured class %s. We will need a mechanism to preserve this between instances.

Reply:
we need to talk about this, I think there are other parameters (throttle?) we might also want to preserve... think for now we should set meaningful default values (after running simulations to understand what they should be).  Then have another ticket for preserving parameter values across restarts.